frappe/core/doctype/access_log/test_access_log.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019, Frappe Technologies and Contributors
+# See license.txt
+
+# imports - standard imports
+from __future__ import unicode_literals
+import json
+import unittest
+
+# imports - module imports
+import frappe
+from frappe.core.doctype.access_log.access_log import make_access_log
+
+
+class TestAccessLog(unittest.TestCase):
+	def test_make_full_access_log(self):
```

Comment:
@gavindsouza

You might have to change the test case. Currently, you are testing if the method is able to create a document or not which is not that useful.

You can write a test to check if the access log gets created when you try to export data...

To do this, you can manually trigger call export_data method and check if the access log has been created with the right required information.

Name of the test case can be `test_access_log_creation_for_data_export`

Similarly, you can write a test case for private file access as well...

Reply:
tests for private file access and exporting csv done